public/app/plugins/panel/graph2/GraphSeriesToggler.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
```

Comment:
We can't use GraphController in Explore because it needs DataFrame[] as data. The GrapSeriesToggler uses GraphSeriesXY instead and can be used by both GraphController and ExploreGraphPanel. Not sure if this component is the correct way to solve this. Not sure about the name as well.